TileFormats/VectorData/README.md

```diff
@@ -1,19 +1,213 @@
+| v | The vertical coordinate of the vertex in the tile. When the v value is 0, the vertex is on the southern edge of the tile. When the value is 32767, the vertex is on the northern edge of the tile. For other values, the vertex's latitude is a linear interpolation between the latitudes of the southern and northern edges of the tile. |
+| height | The height of the vertex of the tile. When the height value is 0, the vertex's height is equal to `MINIMUM_HEIGHT` from the feature table. When the value is 32767, the vertex's height is equal to `MAXIMUM_HEIGHT` from the feature table. For other values, the vertex's height is a linear interpolation of the minimum and maximum heights. |
+
+The values are then delta and ZigZag encoded. The delta encoding ensures the values are small integers. The ZigZag encoding ensures the values are positive integers. Example encoding code is listed below:
+```javascript
+function zigZag(value) {
+    return ((value << 1) ^ (value >> 15)) & 0xFFFF;
+}
+
+var lastU = 0;
+var lastV = 0;
+var lastHeight = 0;
+
+for (var i = 0; i < length; ++i) {
+    var u = uBuffer[i];
+    var v = vBuffer[i];
+    var height = heightBuffer[i];
+    
+    uBuffer[i] = zigZag(u - lastU);
+    vBuffer[i] = zigZag(v - lastV);
+    heightBuffer = zigZag(height - lastHeight);
+
+    lastU = u;
+    lastV = v;
+    lastHeight = height;
+}
+```
+
+Example decoding code is listed below:
+```javascript
+function zigZagDecode(value) {
+    return (value >> 1) ^ (-(value & 1));
+}
+
+var u = 0;
+var v = 0;
+var height = 0;
+
+for (var i = 0; i < length; ++i) {
+    u += zigZagDecode(uBuffer[i]);
+    v += zigZagDecode(vBuffer[i]);
+    height += zigZagDecode(heightBuffer[i]);
+    
+    uBuffer[i] = u;
+    vBuffer[i] = v;
+    heightBuffer[i] = height;
+}
+```
+
+#### Polygon positions
+
+**Figure 3**: Polygon layout.
+
+![](figures/polygon.jpg)
+
+The number of positions for each polygon is determined by the value of its `POLYGON_COUNT`. For example, let the first polygon count be 5. The first polygon's u values start at offset `0` and end at `5 * sizeof(uint16)`. Its v values start at `polygonPositionsByteLength / 2` and end at `polygonPositionsByteLength / 2 + 5 * sizeof(uint16)`.
+
+The positions of the polygons must be the outer ring positions listed in counter-clockwise order.
+
+#### Polyline positions
+
+**Figure 4**: Polyline layout.
+
+![](figures/polyline.jpg)
+
+The number of positions for each polyline is determined by the value of its `POLYLINE_COUNT`. From the first point on the polyline, each successive point creates a segment connected to the previous.
+
+#### Point positions
+
+**Figure 5**: Point layout.
+
+![](figures/point.jpg)
```

Comment:
In the diagram `pointPositionsByteLength / 2` should be `pointPositionsByteLength / 3`